tests/unit_tests/test_wave_02.py

```diff
@@ -23,18 +23,22 @@ def test_get_items_by_category():
     assert item_c in items
     assert item_b not in items
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_get_no_matching_items_by_category():
     item_a = Item(category="clothing")
     item_b = Item(category="clothing")
-    item_c = Item(category="clothing")
+    item_c = Item(category="clothing") 
     vendor = Vendor(
         inventory=[item_a, item_b, item_c]
     )
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+
+    assert vendor.inventory == [item_a, item_b, item_c]
```

Comment:
This new assert works to make sure the inventory was not changed, but we should also have an assert making sure that the returned value `items`, is empty.